src/main/java/io/reactiverse/pgclient/PgConnectOptions.java

```diff
@@ -330,4 +335,31 @@ public PgConnectOptions setTcpQuickAck(boolean tcpQuickAck) {
   public PgConnectOptions setEnabledSecureTransportProtocols(Set<String> enabledSecureTransportProtocols) {
     return (PgConnectOptions) super.setEnabledSecureTransportProtocols(enabledSecureTransportProtocols);
   }
+
+  /**
+   * Initialize with the default options, if env variables are specified they take precedence over these options.
+   */
+  private void init() {
+    if (getenv("PGHOSTADDR") == null) {
+      host = getenv("PGHOST") == null ? DEFAULT_HOST : getenv("PGHOST");
+    } else {
+      host = getenv("PGHOSTADDR");
+    }
+    if (getenv("PGPORT") != null) {
+      try {
+        port = parseInt(getenv("PGPORT"));
+      } catch (NumberFormatException e) {
+        port = DEFAULT_PORT;
```

Comment:
no need for that, port is already set

Reply:
I use a comment to explain what the port will be.